engine/src/cmd/collision.cpp

```diff
@@ -292,15 +292,24 @@ void Collision::collide( Unit* unit1,
     // See commit a66bdcfa1e00bf039183603913567d48e52c7a8e method Unit->jumpReactToCollision for an example
     // I assume this is for "always open" jump gates that you pass through
 
+    collision1.apply_force = false;
+    collision1.deal_damage = false;
+    collision2.apply_force = false;
+    collision2.deal_damage = false;
+
     collision1.shouldApplyForceAndDealDamage(unit2);
     collision2.shouldApplyForceAndDealDamage(unit1);
 
     float elasticity = 0.8f;
-    collision1.applyForce(elasticity, collision2.mass, collision2.velocity);
-    collision2.applyForce(elasticity, collision1.mass, collision1.velocity);
+    if (collision1.apply_force && collision2.apply_force) {
+        collision1.applyForce(elasticity, collision2.mass, collision2.velocity);
+        collision2.applyForce(elasticity, collision1.mass, collision1.velocity);
+    }
 
-    collision1.dealDamage(collision2);
-    collision2.dealDamage(collision1);
+    if (collision1.deal_damage && collision2.deal_damage) {
```

Comment:
Same here. First line of dealDamage checks the variable.
In short, I don't see what your changes actually do to fix the issue. Also, I don't see how the issue is caused based on this. Unless you come up with a better hypothesis, I think the root cause is the function isJumppoint. It should return without setting either apply_force or apply_damage to true.
As this appears to work, we should probably check the data or WC to see what's there.

Reply:
See previous comment, above.

Reply:
You might be right about isJumppoint though. I'll take a look at that as well.

Reply:
Undid this too